src/modules/decklink/consumer_decklink.cpp

```diff
@@ -575,7 +575,7 @@ class DeckLinkConsumer
 					ssize_t size = stride * height;
 
 					// convert lower field first to top field first
-					if ( !progressive )
+					if ( !progressive && m_displayMode->GetFieldDominance() == bmdUpperFieldFirst )
```

Comment:
Keep in mind this NTSC patch fixes playback for allot of native NTSC example video's besides the ones I correctly created. Try the SD25 SD50 and MPEG examples with this f70d47c build and some previous original melt builds.

http://www.gvgdevelopers.com/concrete/products/k2/test_clips/

Footage with DV 4:1:1 lowerfield plays fine with this patch, also rawvideo uyvy422 that plays natively from Blackmagic Design Media Express also plays correct with this version. This is without any XML hacks like progressive=0 top_field_first=1.

How can this be wrong?